src/CalcViewModel/Common/CopyPasteManager.cpp

```diff
@@ -339,6 +340,11 @@ bool CopyPasteManager::ExpressionRegExMatch(vector<wstring> operands, ViewMode m
     return expMatched;
 }
 
+void CopyPasteManager::TrimOperand(std::wstring& operand, size_t exceededBy)
```

Comment:
In general, we want to avoid having methods directly modifying input values.  A safer, more testable, and more standard design is to leave the input alone and return the modified version as the result.  For an example, see [`wstring CopyPasteManager::SanitizeOperand(const wstring& operand)`](https://github.com/Microsoft/calculator/blob/500d84ab5ee0eef5d198012a73f13486662be7f4/src/CalcViewModel/Common/CopyPasteManager.cpp#L410).

Reply:
Will keep that in mind. Thanks.